scripts/init.js

```diff
@@ -80,14 +81,20 @@ module.exports = function(appPath, appName, verbose, originalDirectory) {
     console.log('Success! Created ' + appName + ' at ' + appPath + '.');
     console.log('Inside that directory, you can run several commands:');
     console.log();
-    console.log('  * npm start: Starts the development server.');
-    console.log('  * npm run build: Bundles the app into static files for production.');
-    console.log('  * npm run eject: Removes this tool. If you do this, you can’t go back!');
+    console.log(chalk.cyan('npm start'));
+    console.log('  Starts the development server.');
+    console.log();
+    console.log(chalk.cyan('npm run build'));
+    console.log('  Bundles the app into static files for production.');
+    console.log();
+    console.log(chalk.cyan('npm run eject'));
+    console.log('  Removes this tool and copies build dependencies, configs, and scripts into the app directory.')
```

Comment:
<img width="842" alt="screen shot 2016-09-11 at 16 51 42" src="https://cloud.githubusercontent.com/assets/497214/18417706/379a3a26-7840-11e6-894c-95fdd5d384dd.png">

The new message looks good to me. But let's wrap this line at 80 characters, so it doesn't break with the "standard" terminal window size.